src/glue/jobs/json_to_parquet.py

```diff
@@ -37,42 +37,42 @@
     "enrolledparticipants": ["ParticipantIdentifier"],
     "fitbitprofiles": ["ParticipantIdentifier", "ModifiedDate"],
     "fitbitdevices": ["ParticipantIdentifier", "Date"],
-    "fitbitactivitylogs": ["LogId"],
+    "fitbitactivitylogs": ["ParticipantIdentifier", "LogId"],
     "fitbitdailydata": ["ParticipantIdentifier", "Date"],
-    "fitbitecg": ["FitbitEcgKey"],
+    "fitbitecg": ["ParticipantIdentifier", "FitbitEcgKey"],
     "fitbitintradaycombined": ["ParticipantIdentifier", "Type", "DateTime"],
     "fitbitrestingheartrates": ["ParticipantIdentifier", "Date"],
-    "fitbitsleeplogs": ["LogId"],
-    "healthkitv2characteristics": ["HealthKitCharacteristicKey"],
-    "healthkitv2samples": ["HealthKitSampleKey"],
-    "healthkitv2heartbeat": ["HealthKitHeartbeatSampleKey"],
-    "healthkitv2statistics": ["HealthKitStatisticKey"],
-    "healthkitv2clinicalrecords": ["HealthKitClinicalRecordKey"],
-    "healthkitv2electrocardiogram": ["HealthKitECGSampleKey"],
-    "healthkitv2workouts": ["HealthKitWorkoutKey"],
-    "healthkitv2activitysummaries": ["HealthKitActivitySummaryKey"],
-    "garminactivitydetailssummary": ["ParticipantID", "SummaryId"],
-    "garminactivitysummary": ["ParticipantID", "SummaryId"],
-    "garminbloodpressuresummary": ["ParticipantID", "SummaryId"],
-    "garmindailysummary": ["ParticipantID", "StartTimeInSeconds"],
-    "garminepochsummary": ["ParticipantID", "SummaryId"],
-    "garminhealthsnapshotsummary": ["ParticipantID", "StartTimeInSeconds"],
-    "garminhrvsummary": ["ParticipantID", "StartTimeInSeconds"],
-    "garminmanuallyupdatedactivitysummary": ["ParticipantID", "SummaryId"],
-    "garminmoveiqactivitysummary": ["ParticipantID", "SummaryId"],
-    "garminpulseoxsummary": ["ParticipantID", "SummaryId"],
```

Comment:
Was `ParticipantID` the wrong key?

Reply:
There's a one-to-one mapping from `ParticipantIdentifier` to `ParticipantID`. They are two different ways of representing the same identifier.

`ParticipantIdentifier` is the only field present in every data type (including SymptomLog) and the CE docs suggest it's the more "official" identifier -- kind of like `HealthCode` to `ExternalId` in mPower.